<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameOfLifeTab</class>
 <widget class="QWidget" name="gameOfLifeTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="GolWidget" name="golWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="controlLayout">
     <item>
      <layout class="QHBoxLayout" name="plaeWidthLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Plane width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="planeWidthSpinBox">
         <property name="suffix">
          <string> cells</string>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="planeHeightLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Plane height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="planeHeightSpinBox">
         <property name="suffix">
          <string> cells</string>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="intervallLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Generation intervall</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intervalSpinBox">
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="startStopLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="evolveButton">
         <property name="text">
          <string>Next Generation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="randomizeButton">
       <property name="text">
        <string>Randomize</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="importExportLeyout">
       <item>
        <widget class="QPushButton" name="importButton">
         <property name="text">
          <string>import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>export</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GolWidget</class>
   <extends>QWidget</extends>
   <header>golwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
